src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
'there should always be 4 cards in a row (not 2, 3 or 5)', so please remove this